core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -617,6 +617,7 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     ConditionalTech("after discovering [tech]", UniqueTarget.Conditional),
     ConditionalNoTech("before discovering [tech]", UniqueTarget.Conditional),
 
+    ConditionalFirstCivToAdopt("if no other Civilization has adopted this", UniqueTarget.Conditional),
```

Comment:
Why are we adding an extra unique as part of this change? Sounds like an entirely separate pr